src/11.numberformat.js

```diff
@@ -723,7 +724,7 @@ function ToRawFixed (x, minInteger, minFraction, maxFraction) {
     let idx,
 
         // We can pick up after the fixed formatted string (m) is created
-        m   = Number.prototype.toFixed.call(x, maxFraction),
+        m = decimalAdjust('round', x, -maxFraction).toFixed(maxFraction),
```

Comment:
Thanks a lot for letting me know it, now I could know the process of this project/spec. So, sure thing! I will try to report this thing to the spec repository if this is missing in the spec😃